src/core/lib/surface/init.cc

```diff
@@ -109,6 +109,7 @@ void grpc_init(void) {
       g_shutting_down_cv->SignalAll();
     }
     grpc_iomgr_init();
+    // Note: use AresInit() when we remove this function.
```

Comment:
`// TODO(yijiem): `. This will help us track work, and batch them with the current domain owner.

Reply:
removed